### ISSUE TITLE
Web - Poor Color Contrast of Buttons on Blue Background

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#72]
The color contrast of the black dotted line around the different buttons (Menu, Close, English, Spanish, French, etc.) on the blue background ( #0071BC ) is 4.09:1 and needs to be 4.5:1 or higher to pass. 

IE 11 uses a white dotted line that is as hard to see as the blue one from Firefox.

**Comment by KyleLivingstonDOL** [comment]
Thanks John. The dotted line issue sounds like a browser one rather than an ILAB/DOL one (since it is different in IE 11 versus FF). I'm not sure that is something that we can remedy at this point. ZTI, can you please confirm that this is a browser issue and not a DOL one? If so, John, can we close this out?

**Comment by jfc3-dol** [comment]
There is a way to set the "outline" to have enough color contrast between it and the button.

Here is an example of what I'm talking about in one of my talk examples. If you tab through the page, all the INPUT fields including the buttons at the bottom will get a two or three-pixel outline around it. If you want to check the button either click in the "Note" field at the end and then tab or SHIFT + TAB from the bottom of the page up since it's so long.

http://jfciii.com/presentations/examples/add.html

Below is the CSS I used to make that happen. You might not need all of it or might want to change the colors or widths, but we can talk about it if there are any questions. You need to have hover, focus, and active, so it works in all browsers.

input:hover, input:focus, input:active, select:hover, select:focus, select:active, textarea:hover, textarea:focus, textarea:active 
{ outline: solid 2px #000;}

input[type="checkbox"]:hover, input[type="checkbox"]:focus, input[type="checkbox"]:active, input[type="radio"]:hover, input[type="radio"]:focus, input[type="radio"]:active 
{ outline: solid 3px #000;}

**Comment by trevitalgit** [comment]
added outline to menu button and side navigation buttons.
![image](https://user-images.githubusercontent.com/53573580/63014833-77a44100-be5d-11e9-9485-6c6057c2ee72.png)

**Comment by jfc3-dol** [comment]
It's not perfect, but having the white border and the black outline helps denote the button from the Menu background.

I'm going to close this issue, but we should work on a better way for the next version of the application.
